package.json

```diff
@@ -18,7 +18,9 @@
   },
   "homepage": "https://github.com/leedoughty/mars-rover#readme",
   "dependencies": {
-    "express": "^4.17.1"
+    "axios": "^0.21.0",
```

Comment:
Whoops, don't think this is being used any more :)

Reply:
Ah good spot! 😄 I started looking into using axios originally, but wanted to try out node-fetch instead (as I hadn't used it before), I will remove axios as a dependency now